pkg/webhookutil/multi_error.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhookutil
+
+import "strings"
+
+type MultiError []error
+
+func (m MultiError) Error() string {
```

Comment:
this should be tested. 

btw. are you aware that hashicorp already created such util? https://github.com/hashicorp/go-multierror/tree/master I used that in past and I really like the output that they produce

```
5 errors occurred:
	* foo
	* bar
	* zoo
	* eee
	* ddd
```

output from your Error()
```
foo; bar; zoo; eee; ddd
```